include/update_gcontact.php

```diff
@@ -21,11 +21,13 @@ function update_gcontact_run(&$argv, &$argc){
 
 	$r = q("SELECT * FROM `gcontact` WHERE `id` = %d", intval($contact_id));
 
-	if (!$r)
+	if (!dbm::_is_result($r)) {
 		return;
+	}
 
 	if (!in_array($r[0]["network"], array(NETWORK_DFRN, NETWORK_DIASPORA, NETWORK_OSTATUS)))
 		return;
+	}
```

Comment:
Where does this belongs to?

BTW: Are you running your own instance? I hope that you are doing so and that you have tested your changes on a real life instance for some days?